src/routes/log.js

```diff
@@ -11,6 +11,13 @@ function log(req, res) {
   });
 }
 
-router.get('/', log);
+async function write(req, res) {
```

Comment:
Yes. I haven't done any test work for this as I am waiting to fix the testing bug first.